website/tutorial/part_2/part_2a/04_html_server.md

```diff
@@ -110,138 +115,159 @@ Here is the current architecture of our application:
-  The Exosphere runtime needs this information
-  in order to automatically subscribe the service to these messages.
-  Currently our application doesn't contain any other services
-  that could be communicated with,
-  so this section is empty for now.
-  We'll add some commands here soon!
 
+* The service **type**, **description**, and **author**
+* The **startup** section defines how to boot up the service.
+  * the **online-text** section contains the string to look for in the terminal
+    output to determine when the service has successfully started up. The
+    Exosphere runtime only sends traffic to fully available instances.
+* The **messages** section lists all the messages that this service will send
+  and receive. The Exosphere runtime needs this information in order to
+  automatically subscribe the service to these messages. Currently our
+  application doesn't contain any other services that could be communicated
+  with, so this section is empty for now. We'll add some commands here soon!
 
 The other files in this directory are just a normal
-[ExpressJS](http://expressjs.com)
-application.
+[ExpressJS](http://expressjs.com) application.
 
+## Booting up the application
 
-## Setting up the service
-
-With all files in place,
-the Exosphere CLI has all the information to set up our application.
-Let's check that the overall configuration is correct,
-and have Exosphere set up the service for us:
+With all files in place, the Exosphere CLI has all the information to set up our
+application. Let's have Exosphere run our application for us:
 
 ```
-$ exo setup
+exo run
 ```
 
-We see how it uses Node's package management system (NPM)
-to download and install
-the external [ExpressJS](http://expressjs.com) and [Pug](http://jade-lang.com/) (formerly Jade) modules for us,
-so that the service is ready to run.
 The output should look something like:
 
 ```
-Setting up Todo application 0.0.1
-
-  exo-setup  starting setup of 'html-server'
-html-server  /Users/kevin/exosphere-sdk/tutorial/part_2/code_02/todo-app/html-server
-├── express@4.13.4
-└── jade@1.11.0
-html-server  PROCESS ENDED
-html-server  EXIT CODE: 0
-  exo-setup  setup of 'html-server' finished
-  exo-setup  setup complete
-```
-
-
-## Booting up the application
-
-To test that everything works, let's check that the application boots up:
-
-```
-$ exo run
+>>> docker-compose --file run_development.yml up --build
+Creating network "todoapp_default" with the default driver
+Pulling exocom0.26.1 (originate/exocom:0.26.1)...
+0.26.1: Pulling from originate/exocom
+add3ddb21ede: Pull complete
+7b0e69015745: Pull complete
+Digest: sha256:56e176f25e21732ccad671375fba8177a04a3fc7ea8bc1decf9e715c6af961c6
+Status: Downloaded newer image for originate/exocom:0.26.1
+Building web-service
+Step 1/6 : FROM node
+latest: Pulling from library/node
+85b1f47fba49: Pull complete
+ba6bd283713a: Pull complete
+817c8cd48a09: Pull complete
```

Comment:
This output is too lengthy. Can you please eliminate some repeated lines, like these ones here showing `xxxx: Pull complete`? Maybe keep only 2 to show there can be multiple ones...